servercom/servercom.go

```diff
@@ -0,0 +1,138 @@
+package serverCom
+
+import (
+	"fmt"
+	mm "github.com/mattermost/platform/model"
+)
+/*
+ServerCom acts as a mitigator between the frontend and the mattermost model API.
+*/
+type ServerCom struct {
+	Client mm.Client
+	Team mm.Team
+	Channel *mm.Channel
+}
+
+/*
+Startup accepts the url and login credentials for a user, and returns a new serverCom struct.
+*/
+func Startup(url string, username string, password string) ServerCom {
```

Comment:
Sorry for the delayed response, but yeah I do like these methods. However, we also need some way to tell whenever a new message is available. The screen only updates whenever an event happens, so if there is a new message that was sent, you won't see the message until you press a key.

There are two ways that I think we can handle this though, we can either have a timer where the screen gets updated every so often or we can have the server send everyone's app a message saying that there is a new message available and the screen needs to be updated to reflect that change.